.github/workflows/drstrange_deployment.yml

```
@@ -0,0 +1,52 @@
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  staging_deploy:
+    if: (github.event_name != 'pull_request') || (github.event_name == 'pull_request' && github.event.action == 'labeled' && github.event.label.name == 'drstrange_deploy')
+    runs-on: ubuntu-latest
+    environment: drstrange
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)